addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBaseActuatorHandler.java

```diff
@@ -220,23 +217,22 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
 
         try {
-            EEP eep = EEPFactory.createEEP(sendingEEPType);
             Configuration channelConfig = channel.getConfiguration();
 
-            // Eltako rollershutter do not support absolute value just values relative to the current position
-            // If we want to go to 80% we must know the current position to determine how long the rollershutter has
-            // to drive up/down. However items seems to be stateless, so we have to store the state by ourself.
-            // The currentState is updated by EnOceanBaseSensorHandler after receiving a response.
-            State currentState = getCurrentState(channelId);
+            EEP eep = EEPFactory.createEEP(sendingEEPType);
+            eep.convertFromCommand(channelId, channelTypeId, command, channelState, channelConfig);
 
-            ESP3Packet msg = eep
-                    .setSenderId(senderId).setDestinationId(destinationId).convertFromCommand(channelId,
-                            channel.getChannelTypeUID().getId(), command, currentState, channelConfig)
-                    .setSuppressRepeating(getConfiguration().suppressRepeating).getERP1Message();
+            if (eep.hasData()) {
+                ESP3Packet msg = eep.setSenderId(senderId).setDestinationId(destinationId)
+                        .setSuppressRepeating(getConfiguration().suppressRepeating).getERP1Message();
+
+                getBridgeHandler().sendMessage(msg, null);
+            } else {
+                logger.debug("Not gonna send this message...");
```

Comment:
Not very helpful to a user, is it?

Reply:
It helped me verifying that the code worked :-)
Removed it.